server/controllers/finance/reports/clientsReport/index.js

```diff
@@ -24,25 +26,47 @@ function document(req, res, next) {
   const session = {};
   let report;
 
-
   _.defaults(session, {
-    dateFrom : new Date(params.dateFrom),
-    dateTo : new Date(params.dateTo),
-    detailPrevious : params.detailPrevious,
-    ignoredClients : params.ignoredClients,
-    enterprise : req.session.enterprise,
+    dateFrom: new Date(params.dateFrom),
+    dateTo: new Date(params.dateTo),
+    detailPrevious: params.detailPrevious,
+    ignoredClients: params.ignoredClients,
+    simplePreview: params.simplePreview,
+    enterprise: req.session.enterprise,
+    periodZero : 0,    
   });
 
-  _.defaults(params, { user : req.session.user });
+  _.defaults(params, { user: req.session.user });
+  session.simplePreview = session.simplePreview === 'true';
 
   try {
+    const TEMPLATE = session.simplePreview ? TEMPLATE_SIMPLE : TEMPLATE_COMPLEX;
     report = new ReportManager(TEMPLATE, req.session, params);
   } catch (e) {
     return next(e);
   }
 
-  // Getting data to be rendered
-  return fetchClientsData(session)
+  // if there is some client to ignore
+  if (session.ignoredClients) {
+    const ignoredClients = (Array.isArray(session.ignoredClients)) ? session.ignoredClients : [session.ignoredClients];
```

Comment:
It just seems like a hack and Array.isArray() is semantically clearer.  I suppose you can add it if you want, but it would probably needed a comment like "Ensure ignored clients is an array" or something.